Ports/openssl/package.sh

```diff
@@ -1,15 +1,21 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=openssl
-branch='1.0.2'
-version="${branch}t"
+branch='1.1.1'
+version="${branch}k"
 useconfigure=true
 configscript=Configure
-files="https://ftp.openssl.org/source/old/${branch}/openssl-${version}.tar.gz openssl-${version}.tar.gz 14cb464efe7ac6b54799b34456bd69558a749a4931ecfd9cf9f71d7881cac7bc"
+files="https://ftp.openssl.org/source/openssl-${version}.tar.gz openssl-${version}.tar.gz 892a0875b9872acd04a9fde79b1f943075d5ea162415de3047c327df33fbaee5"
```

Comment:
I believe that when a new release comes out, this file is moved to `/old/` breaking the download. Not sure what the solution would be except for continuously updating this port though :-)

Reply:
You could use another source such as https://ftp.nluug.nl/security/openssl/ to download the file - they don't apply the same directory structure.

Reply:
I'll go with that domain, I've seen it before with GNU ftp mirror :)